modules/alloy/src/main/ts/ephox/alloy/positioning/view/Transitions.ts

```diff
@@ -31,7 +31,7 @@ const shouldApplyTransitionCss = (transition: Transition, decision: RepositionDe
 
 const hasChanges = (position: PositionCss, intermediate: Record<TransitionProp, Optional<string>>): boolean => {
   // Round to 3 decimal points
-  const round = (value: string) => parseFloat(value).toPrecision(3);
+  const round = (value: string) => parseFloat(value).toFixed(3);
```

Comment:
Should we add a test to cover the `toPrecision` -> `toFixed` changes?

Reply:
I'm fine not including it atm, as testing that would be pretty involved/tricky and if it broke anything bad the existing tests would have caught it.